app/views/AddSectionsAgentView.scala.html

```diff
@@ -42,7 +44,7 @@
 
         @p(messages("addSections.completedSections", vm.completedCount), extraClasses = Seq("govuk-!-margin-bottom-7"))
 
-        @taskList(List(("", vm.sections)))
+        @govukTaskList(TaskList((vm.sections)))
```

Comment:
I've noticed `TaskList((vm.sections)) `in a couple of places, is that correct or should it be `TaskList(vm.sections)`?